core/rio/api/src/main/java/org/eclipse/rdf4j/rio/helpers/XMLWriterSettings.java

```diff
@@ -46,6 +46,29 @@
 	public static final RioSetting<Boolean> INCLUDE_ROOT_RDF_TAG = new BooleanRioSetting(
 			"org.eclipse.rdf4j.rio.include_root_rdf_tag", "Include Root RDF Tag", Boolean.TRUE);
 
+	/**
+	 * Boolean setting for RDF/XML Writer to determine if single quotes are used to quote attribute values. By default
+	 * double quotes are used.
+	 * <p>
+	 * Defaults to false.
+	 * <p>
+	 * Can be overridden by setting system property {@codeorg.eclipse.rdf4j.rio.use_single_quotes}
```

Comment:
Shouldn't there be a space between `@code` and `org...`? Confused because github syntax highlighting seems to recognize it, and you're doing it consistently.

Reply:
Ah yes indeed, I copy/pasted the text from the existing settings, and they also don't use a space... So I guess it just works by accident...

Reply:
fixed